app/containers/message/index.tsx

```diff
@@ -401,85 +425,96 @@ class MessageContainer extends React.Component<IMessageContainerProps, IMessageC
 		const canTranslateMessage = autoTranslateRoom && autoTranslateLanguage && autoTranslateMessage !== false && otherUserMessage;
 
 		return (
-			<MessageContext.Provider
-				value={{
-					id,
-					rid,
-					user,
-					baseUrl,
-					onPress: this.onPressAction,
-					onLongPress: this.onLongPress,
-					reactionInit: this.reactionInit,
-					onErrorPress: this.onErrorPress,
-					replyBroadcast: this.replyBroadcast,
-					onReactionPress: this.onReactionPress,
-					onEncryptedPress: this.onEncryptedPress,
-					onDiscussionPress: this.onDiscussionPress,
-					onReactionLongPress: this.onReactionLongPress,
-					onLinkPress: this.onLinkPress,
-					onAnswerButtonPress: this.onAnswerButtonPress,
-					jumpToMessage,
-					threadBadgeColor,
-					toggleFollowThread,
-					replies,
-					translateLanguage: canTranslateMessage ? autoTranslateLanguage : undefined
-				}}
-			>
-				{/* @ts-ignore*/}
-				<Message
-					id={id}
-					msg={message}
-					md={md}
-					rid={rid}
-					author={u}
-					ts={ts}
-					type={t}
-					attachments={attachments}
-					blocks={blocks}
-					urls={urls}
-					reactions={reactions}
-					alias={alias}
-					avatar={avatar}
-					emoji={emoji}
-					timeFormat={timeFormat}
-					style={style}
-					archived={archived}
-					broadcast={broadcast}
-					useRealName={useRealName}
-					isReadReceiptEnabled={isReadReceiptEnabled}
-					unread={unread}
-					role={role}
-					drid={drid}
-					dcount={dcount}
-					dlm={dlm}
-					tmid={tmid}
-					tcount={tcount}
-					tlm={tlm}
-					tmsg={tmsg}
-					fetchThreadName={fetchThreadName}
-					mentions={mentions}
-					channels={channels}
-					isIgnored={this.isIgnored}
-					isEdited={(editedBy && !!editedBy.username) ?? false}
-					isHeader={this.isHeader}
-					isThreadReply={this.isThreadReply}
-					isThreadSequential={this.isThreadSequential}
-					isThreadRoom={!!isThreadRoom}
-					isInfo={this.isInfo}
-					isTemp={this.isTemp}
-					isEncrypted={this.isEncrypted}
-					hasError={this.hasError}
-					showAttachment={showAttachment}
-					getCustomEmoji={getCustomEmoji}
-					navToRoomInfo={navToRoomInfo}
-					handleEnterCall={handleEnterCall}
-					blockAction={blockAction}
-					highlighted={highlighted}
-					comment={comment}
-					isTranslated={isTranslated}
-					isPreview={isPreview}
-				/>
-			</MessageContext.Provider>
+			<PanGestureHandler
```

Comment:
You'll have to evaluate and show us if this new gesture is impacting the other buttons inside Message.